x/dex/types/tick_liquidity.go

```diff
@@ -1,17 +1,73 @@
 package types
 
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
+func (t TickLiquidity) TokenIn() string {
```

Comment:
Imo this approach ends up adding more code than simply moving these fields to the TickLiquidity object and having normal accessors.

Reply:
See above comment